_projects/mcfd_cmp_project.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page_project
+title: Comparison of Meshing and CFD Methods for Accurate Flow Simulations on HPC systems
+date: 2016-03-02
+updated: 2016-03-02
+navbar: Research
+subnavbar: Projects
+project_url:
+status: starting
+topics:
+  - apps
+  - numerics
+keywords:
+  - meshing
+  - CFD
+  - accuracy analysis
+  - performance analysis
+head: lintermann_a
+members:
+  - tsubokura_m
+---
+
+## Research topic and goals
+
+The expertise of the collaborators at the two centers AICS and JSC
```

Comment:
We could, but I tell you: if, IF anyone will upload anything, they will
not care anyway..

On 02.03.16 14:56, Torbjörn Klatt wrote:
> In _projects/mcfd_cmp_project.md
> <https://github.com/JLESC/jlesc.github.io/pull/146#discussion_r54724875>:
> 
>> +topics:
>> +  - apps
>> +  - numerics
>> +keywords:
>> +  - meshing
>> +  - CFD
>> +  - accuracy analysis
>> +  - performance analysis
>> +head: lintermann_a
>> +members:
>> +  - tsubokura_m
>> +---
>> +
>> +## Research topic and goals
>> +
>> +The expertise of the collaborators at the two centers AICS and JSC
> 
> could we agree on one sentence per line?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JLESC/jlesc.github.io/pull/146/files#r54724875>.
>